spec/javascripts/enterprise_admin_spec.js

```diff
@@ -0,0 +1,71 @@
+var enrollmentApiRoot = 'http://localhost/'
+var gettext = function(arg){
```

Comment:
usually we are following to add space before opening braces in javascript files. better to follow the convention in all over places below too.